web/src/lib/components/NostrChat.svelte

```diff
@@ -181,6 +183,11 @@
         }
 
         await updateNostrProfiles();
+
+        while ($user === null) {
```

Comment:
what are you trying to do here? wait until the user has logged in?

i think this is not the right way to do it.

`user` is a "store" that you can "subscribe" to (don't forget to unsubscribe in onDestroy)! see examples of ".subscribe" in our code.

Reply:
No, I'm waiting until $user is available. It seems it's loaded asynchronously by another component, so sometimes it's `null` and you have to wait a little until it contains the actual user. I did the same in `Login.svelte`.

Reply:
Yes, but I think this is the the best way to do it. `user` is a Svelte store, which has .subscribe, so you can be notified when it changes (and check whether it changed to non-null).

Reply:
Done. Give it a look.